__tests__/app.test.js

```diff
@@ -103,4 +103,57 @@ describe("/api/articles", () => {
 				expect(body.articles).toBeSortedBy("created_at", {descending: true});
 			});
 	});
-})
\ No newline at end of file
+})
+
+
+describe("/api/articles/:article_id/comments", () => {
+	it("GET: 200 - responds with a correct comments array", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body }) => {
+				body.comments.forEach((comment) => {
+					expect(typeof comment.comment_id).toBe("number");
+					expect(typeof comment.author).toBe("string");
+					expect(typeof comment.article_id).toBe("number");
+					expect(typeof comment.body).toBe("string");
+					expect(typeof comment.created_at).toBe("string");
+					expect(typeof comment.votes).toBe("number");
+					
+				});
+		});
+	})
+	it("GET: 200 - responds with an array of comments in the descending order", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body }) => {
+				expect(body.comments).toBeSortedBy("created_at", {descending: true});
+			});
+	});
+	it("GET: 200 - responds with en empty arrat when given an id for the existing article, but there is no comments", () => {
```

Comment:
Very nice edge case to check for -
 also there's a slight typo in your test description here